peekaboo/toolbox/ms_office.py

```diff
@@ -59,3 +59,34 @@ def has_office_macros(office_file):
     except Exception as error:
         logger.exception(error)
         return False
+
+def has_office_macros_with_auto_action(office_file):
```

Comment:
This function is very similar to the classic `has_office_macros` detector. Should we perhaps extend the classic rule to accept/expose filter options instead of introducting a new, again highly-specific rule? E.g. we could have whitelist and blacklist parameters as with the other rules in `ruleset.conf`. The classic behaviour would be implemented as:
```
[office_macros]
blacklist.1 : .*
```
and the new rule would be:
```
[office_macros]
blacklist.1 : AutoOpen
blacklist.2 : AutoClose
```
The user-visible benefit would be to be able to detect other macro functions without code changes just by adjusting `ruleset.conf`.

Reply:
Config added to new rule but old rule not yet replaced by new functionality in commit 6a3ba2803d3f077b836d9755fb46e273bdb912b0. Is that intentional?

Reply:
Superseeded by latest changes.